src/main/java/frc/robot/commands/ArmCommand.java

```diff
@@ -0,0 +1,92 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.commands;
+
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.CANSparkMaxLowLevel.MotorType;
+
+import com.revrobotics.AbsoluteEncoder;
+import com.revrobotics.SparkMaxAbsoluteEncoder;
+import com.revrobotics.CANSparkMax.ControlType;
+
+import edu.wpi.first.math.controller.PIDController;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.subsystems.ArmSubsystem;
+import frc.robot.subsystems.SwerveSubsystem;
+
+public class ArmCommand extends CommandBase {
+  double kP = 0;
+  double kI = 0;
+  double kD = 0;
+  double integralMinimum = -0.5;
+  double integralMaximum = 0.5;
+
+  PIDController pid = new PIDController(kP, kI, kD);
+
+  double NEO_GEAR_RATIO = 7.67;
+  double L1 = 0; //length of arm 1
+  double L2 =0; //length of arm 2
+  double z = 0;//constants --> first and second z positions - depends on the height of the node we are going for
+  double nodeX = 0; //the x positiion of the node we are going for
+  double armSpeed = 0.2;
+
+
+  /** Creates a new ArmExtend. */
+  
+  public ArmCommand() {
```

Comment:
The constructor can have an input argument for which level of the node the arm is aiming for.
Then map this input argument to z value and x coordinate, i.e. nodeX.